posthog/settings/sentry.py

```diff
@@ -68,6 +69,8 @@ def sentry_init() -> None:
         sentry_sdk.utils.MAX_STRING_LENGTH = 10_000_000
         # https://docs.sentry.io/platforms/python/
         sentry_logging = sentry_logging = LoggingIntegration(level=logging.INFO, event_level=None)
```

Comment:
```suggestion
        sentry_logging = LoggingIntegration(level=logging.INFO, event_level=None)
```

@macobo is ` sentry_logging = sentry_logging = ` a typo or some python magic I don't understand 😊

Reply:
Don't blame me, blame "Revert: "Revert: "Proposal: " :joy: